src/components/App/actions.js

```diff
@@ -1,9 +1,6 @@
-import {
-	SET_USERS,
-	SET_SLACK_USER_FETCH_ERROR,
-	SET_CURRENT_USER,
-	SET_PREDICTION
-} from './actionTypes';
+import {SET_USERS, SET_SLACK_USER_FETCH_ERROR} from './actionTypes';
+
+import {loadUsers as reallyLoadUsers} from './../../utils/slack';
```

Comment:
Think this is my bad, we can probably think of a better name than reallyLoadUsers - I just wasn't creative yesterday

Maybe something like `fetchUsers`?

```
fetchUsers()
  .then(users => ...)
  .catch(err => ...)
```